tenzir/integration/tests.yaml

```diff
@@ -586,10 +583,12 @@ tests:
 
   Process Query For Field With Skip Attribute:
     tags: [import, export]
+    fixture: ServerTester
     steps:
-      - command: -N import --schema-file="@./misc/schema/zeek-with-skip.schema" zeek-json
+      - command: import -b --schema-file="@./misc/schema/zeek-with-skip.schema" zeek-json
```

Comment:
Is this instance of `--schema-file` a leftover or intentional?

Reply:
`zeek-json` (and all other formats not explicitly listed in the PR description) are not changed here.

Reply:
But `--schema-file` is an option to `import`, not for the specific format. What happens if we specify `import --schema-file=foo json`, is it silently ignored?

Reply:
Yes. For `json`, it is ignored, for `zeek-json` it is not ignored.

(And I think that's okay, given that the whole command is deprecated and to-be-removed soon)